app/v2/notifications/post_notifications.py

```diff
@@ -290,7 +289,11 @@ def save_email_to_queue(
         data
     )
 
-    save_api_email.apply_async([encrypted], queue=QueueNames.SAVE_API_EMAIL)
+    if notification_type == EMAIL_TYPE:
+        save_api_email.apply_async([encrypted], queue=QueueNames.SAVE_API_EMAIL)
+    else:
```

Comment:
Optional: Could check explicitly that notification type is SMS and then fall back to raising an exception if it's not for some reason. Just in case we were ever to call this with a letter accidently